retnet/modeling_retnet.py

```diff
@@ -382,36 +607,34 @@ def forward(
         output_retentions: Optional[bool] = False,
     ) -> Tuple[torch.FloatTensor, torch.FloatTensor, Optional[torch.FloatTensor]]:
         B, T, H = hidden_states.size()
-        (sin, cos), decay_mask = rel_pos
+        (sin, cos), decay_mask = rel_pos[:2]
```

Comment:
This is a design choice. But I prefer unpacking all and handling unneeded stuff later.